amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/reports/Reports.java

```diff
@@ -995,4 +998,13 @@ public String exportToMap(
     public Class getErrorsClass() {
         return ReportErrors.class;
     }
+
+    @GET
+    @Path("/ampTeam/{id}")
+    @Produces(MediaType.APPLICATION_JSON + ";charset=utf-8")
+    @ApiOperation("Get AmpTeam specification")
+    @ApiMethod(ui=false, id = "ampTeam", authTypes = AuthRule.IN_ADMIN)
+    public final AmpTeam getWorkspace(@PathParam("id") Long id) {
+        return TeamUtil.getAmpTeam(id);
+    }
```

Comment:
Since we already have an endpoint to return workspace settings in security, lets move it there to keep things related to workspaces in the same endpoint